lib/handler/file.c

```diff
@@ -26,11 +26,18 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <sys/stat.h>
+#include <sys/types.h>
 #include <time.h>
 #include <unistd.h>
+
 #include "h2o.h"
 
 #define MAX_BUF_SIZE 65000
+#define BOUNDARY_SIZE 20
+#define FIXED_PART_SIZE 49
+/* = size of five "\r\n"s + size of "--" 
+ *   + size of header names w/ column and space (Content-Range and Content-Type) 
+ *   + size of "bytes " and "-" and "/" */
```

Comment:
Could the definition be more expressive (instead of using comments)?

It would be great if you could write like `sizeof("\r\n--") - 1 + BOUNDARY_SIZE + sizeof("\r\nContent-Range: bytes=-/\r\nContent-Type: \r\n\r\n") - 1` so that it would be easier to understand.